src/nyc_trees/apps/home/tests.py

```diff
@@ -2,3 +2,96 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 from __future__ import division
+
+from django.test import TestCase
+from django.http import Http404
+from django.conf import settings
+
+from django.contrib.flatpages.models import FlatPage
+
+from django_tinsel.utils import decorate as do
+
+from apps.core.models import User
+from apps.core.test_utils import make_request
+from apps.home.training.decorators import mark_user, render_flatpage
+
+
+class TrainingTrackingTest(TestCase):
```

Comment:
I wonder if it would be worthwhile to test that a user can actually make it all the way through the training process. Maybe in a future PR?